src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -391,7 +392,12 @@ class LicenceService(
     val submitter = communityOffenderManagerRepository.findByUsernameIgnoreCase(username)
       ?: throw ValidationException("Staff with username $username not found")
 
-    val updatedLicence = licenceEntity.submit(submitter)
+    val updatedLicence = when (licenceEntity) {
+      is CrdLicence -> licenceEntity.submit(submitter)
+      is VariationLicence -> licenceEntity.submit(submitter)
+      is HardStopLicence -> TODO("Submitting hard stop licences not supported yet")
```

Comment:
should have some validation here - ensure correct type of user per licence and load up the correct DB record to associate them with